package.json

```diff
@@ -184,7 +184,8 @@
             "fortran_fixed-form": "cd $dir && gfortran $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             "fortran": "cd $dir && gfortran $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             "sml": "cd $dir && sml $fileName",
-            "mojo": "mojo run"
+            "mojo": "mojo run",
+            "erlang": "cd $dir && erlc $fileName && erl -noshell -s $fileNameWithoutExt start -s init stop"
```

Comment:
It seems that `start` is a dynamic value? Any better way to improve this?

Reply:
I use `escript` to take the place of this command. Since Erlang use VM and we can specify the entrance function, the guide I'm learning uses `start` as entrance. However, I found that I can use [escript](https://www.erlang.org/doc/man/escript.html) to run short Erlang programs and this program use `main` as entrance. In other words, I can use `erlc $filename && erl -noshell -s $fileNameWithoutExt main -s init stop`, which maybe equals to `escript $filename`. But it seems that `escript` is the best way solve this problem, so I use it at last.

Reply:
![屏幕截图_20240311_143310](https://github.com/formulahendry/vscode-code-runner/assets/49590319/91c3aa71-07d6-43db-94ae-4939355f42d2)